package.json

```diff
@@ -28,30 +28,42 @@
   ],
   "scripts": {
     "docs": "jsdoc -c .jsdoc.js",
-    "lint": "eslint '**/*.js'",
-    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "system-test": "mocha system-test/*.js --timeout 600000",
-    "test": "c8 mocha",
-    "fix": "eslint '**/*.js' --fix",
+    "lint": "gts fix && eslint --fix samples/*.js",
+    "samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../",
+    "system-test": "mocha build/system-test",
+    "test": "c8 mocha build/test",
+    "fix": "gts fix",
     "docs-test": "linkinator docs",
     "predocs-test": "npm run docs",
-    "prelint": "cd samples; npm link ../; npm i"
+    "clean": "gts clean",
+    "compile": "tsc -p . && cp -r protos build/",
+    "compile-protos": "compileProtos src",
+    "prepare": "npm run compile"
   },
   "dependencies": {
-    "google-gax": "^1.7.5"
+    "google-gax": "^1.14.1"
   },
   "devDependencies": {
-    "c8": "^7.0.0",
-    "codecov": "^3.0.0",
+    "@types/mocha": "^5.2.5",
+    "@types/node": "^12.12.27",
+    "c8": "^7.1.0",
+    "codecov": "^3.6.5",
     "eslint": "^6.0.0",
     "eslint-config-prettier": "^6.0.0",
     "eslint-plugin-node": "^11.0.0",
     "eslint-plugin-prettier": "^3.0.0",
+    "gts": "^1.0.0",
     "jsdoc": "^3.5.5",
     "jsdoc-fresh": "^1.0.1",
     "jsdoc-region-tag": "^1.0.2",
     "linkinator": "^2.0.0",
-    "mocha": "^7.0.0",
-    "prettier": "^1.7.4"
+    "mocha": "^7.0.1",
+    "null-loader": "^3.0.0",
+    "pack-n-play": "^1.0.0-2",
+    "prettier": "^1.13.7",
+    "ts-loader": "^6.2.1",
+    "typescript": "^3.7.0",
```

Comment:
@bcoe are we safe to go with 3.7 here now?  Can't remember 😆

Reply:
so, I think one of the main reasons people were yelling at us was that angular was on older TypeScript versions; they've now updated.

I think we should start cautiously moving forward, keeping in mind this bit some people.

Reply:
What is the safe version I could use for typescript here. Do we have runbook to follow when upgrade package.json?

Reply:
Agreed - if Angular uses 3.7 we can try this as well. We'll downgrade if folks start complaining.